File: 568/step_3.pl
#!/usr/bin/perl -w

use warnings;
use strict;

chomp(my $n = <STDIN>);
my $text = "You have chosen a ";
if ( $n == 1 ) {
    $text .= "square";
} elsif ( $n == 2 ) {
    $text .= "circle";
} elsif ( $n == 3 ) {
    $text .= "triangle";
} elsif ( $n == 4 ) {
    $text .= "rhombus";
} else {
    $text = "There is no such shape!";
}

print $text;

__END__

=head1 NAME

Shape

=head1 DESCRIPTION

Write a program, which reads the number of the shape (1 – square, 2 – circle, 3 – triangle, 4 – rhombus) and prints the text "You have chosen a square" (or circle, or triangle, or rhombus, depending on the number). If it is a number, which doesn't correspond to any of the listed shapes, the program should output: "There is no such shape!".

Note: output text should exactly match the sample! Including letters case and location of spaces.

Sample Input:

1

Sample Output:

You have chosen a square

=cut

File: 67/3366/step_3.py
a = int(input())
b = int(input())
c = int(input())
d = int(input())

print("\t", end='')
for i in range(c, d+1):
    print(i, end="\t")
print()

for x in range (a, b+1):
    print(x, end="\t")
    for y in range (c, d+1):
        print(x * y, end="\t")
    print()
